src/components/Calculator.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
+  const [result, setResult] = useState('0');
+
+  const handleClick = (value) => {
+    setResult((prevResult) => {
+      if (prevResult === '0') {
+        return value;
+      }
+      return prevResult + value;
+    });
+  };
+
+  const handleCalculate = () => {
+    try {
+    //   eslint-disable-next-line
+      setResult(eval(result).toString());
+    } catch (error) {
+      setResult('Error');
+    }
+  };
+
+  const handleClear = () => {
+    setResult('0');
+  };
+
+  return (
+    <div className="calculator-container">
+      <div className="calculator">
+        <input type="text" value={result} readOnly className="result-input w-100" />
+        <div className="keypad">
+          <div className="row">
+            <div className="col">
+              <button type="button" onClick={handleClear} className="button-gray w-100">AC</button>
+            </div>
+            <div className="col">
+              <button type="button" onClick={() => handleClick('+/-')} className="button-orange w-100">+/-</button>
+            </div>
+            <div className="col">
```

Comment:
> In Calculator.js, create a React component that matches this design:
- [ ] You have done great with styling your calculator application. :+1: :+1:  but you were given a design by MIcroverse, I'll suggest you stick to that as that's part of the requirement. :+1: :+1: :+1: